Formula/ice.rb

```diff
@@ -87,10 +89,25 @@ def install
       end
     end
 
-    cd "php" do
-      args << "install_phpdir=#{share}/php"
-      args << "install_libdir=#{lib}/php/extensions"
-      system "make", "install", *args
+    if build.with? "php"
+      cd "php" do
+        args << "install_phpdir=#{share}/php"
+        args << "install_libdir=#{lib}/php/extensions"
+        system "make", "install", *args
+      end
+    end
+
+    if build.with? "python"
+      python = PythonRequirement.new
```

Comment:
don't need to use this once the `depends_on` is done above as it'll be first in the PATH.